### PR TITLE
Fix windows separators in pathStr

### PR DESCRIPTION
**Pull request by Guria**
follow up from https://github.com/bem/webpack-bem-loader/issues/26

### CODE REVIEW

**Comment by Guria** [comment]
hmm. This will fix an issue but seems problem is not in require and/or webpack. investigating.

**Comment by Guria** [comment]
See https://github.com/bem/webpack-bem-loader/issues/26#issuecomment-322731066

File: index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 var path = require('path');
 var assert = require('assert');
-
+// path.sep shouldn't be used since require doesn't support windows separator
```
Comment (Guria) [inline index.js:3]: comment statement isn't true.
see https://github.com/bem/webpack-bem-loader/issues/26#issuecomment-322731066

**Comment by Guria** [comment]
@Yeti-or How about a `keepSeparator` flag? Do not replace anything and use `path.sep` when making relative if flag is set to true or replace any windows separators with nix ones otherwise?

**Comment by Yeti-or** [comment]
I will check it at tonight, thanx

**Comment by Guria** [comment]
@Yeti-or Did you come up with smth?

**Comment by tadatuta** [comment]
@yeti-or ping!

**Comment by Guria** [comment]
would be great to hear smth here so we would be able to move on

**Comment by Yeti-or** [comment]
We fixed it inside `webpack-bem-loader` it's not a problem with `required-path`

**Comment by Yeti-or** [comment]
@Guria check https://github.com/bem/webpack-bem-loader/pull/52
I am very sorry for such a long delay 😔

**Pull request closed without merging**